Add AYON APIs & Resources section (#239)
ynput/ayon-documentation: website/docs/dev_api_cpp.md

---
id: dev_api_cpp
title: AYON C++ API
sidebar_label: C++ API
description: Information about AYON C++ API
toc_max_heading_level: 5
---

> What is the C++ Language standard? it's ISO C++17 Standard

Reviewer: Having this sentence on first line seems weird (out of context).

Author: https://github.com/ynput/ayon-documentation/pull/239/commits/b534281136c835274d3343536b40171be9f9bca0

## About AYON C++ API

The AYON CPP API allows communication with the AYON server.

> Its focus is on communication with the server Resolve endpoint.

Reviewer: ```suggestion
Its focus is on communication with the server resolve endpoint.
```

> Its focus is on communication with the server Resolve endpoint.
> and it's meant to be used mainly by [AYON USD Resolver](dev_api_usd_resolver.md).

Reviewer: ```suggestion
Implementation is mainly focused on [AYON USD Resolver](dev_api_usd_resolver.md).
```

Author: https://github.com/ynput/ayon-documentation/pull/239/commits/b534281136c835274d3343536b40171be9f9bca0

## Get Started

:::tip Development Environment

> We assume you already had your c++ dev environment setup as well as cmake.

Reviewer: ```suggestion
We assume you already have setup your c++ dev environment as well as cmake.
```

Author: https://github.com/ynput/ayon-documentation/pull/239/commits/b534281136c835274d3343536b40171be9f9bca0

:::

### Fetch and Build The library

> AYON CPP API is a static lib so you'd need to build it first.
> 
> You can use the provided python script to do the hard lifting for you.

Reviewer: ```suggestion
AYON CPP API is a static lib.

You need to build it first. We've prepared python script to do the hard lifting for you.
```

Author: https://github.com/ynput/ayon-documentation/pull/239/commits/b534281136c835274d3343536b40171be9f9bca0
